### ISSUE TITLE
Err 500 on api threads/{id}

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#1]
just got this error for routes like `/threads/{id}`

```
api_1 | 2016-03-14 06:59:44,542 DEBUG [caliopen.api.user.authentication][waitress] Authentication via Access Token
api_1 | 2016-03-14 06:59:44,545 ERROR [waitress][waitress] Exception when serving /api/v1/threads/4078e616-390c-478a-abb2-dddaae7c10f0
api_1 | Traceback (most recent call last):
api_1 |   File "/usr/local/lib/python2.7/dist-packages/waitress-0.8.10-py2.7.egg/waitress/channel.py", line 336, in service
api_1 |     task.service()
api_1 |   File "/usr/local/lib/python2.7/dist-packages/waitress-0.8.10-py2.7.egg/waitress/task.py", line 169, in service
api_1 |     self.execute()
api_1 |   File "/usr/local/lib/python2.7/dist-packages/waitress-0.8.10-py2.7.egg/waitress/task.py", line 388, in execute
api_1 |     app_iter = self.channel.server.application(env, start_response)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/pyramid-1.5.8-py2.7.egg/pyramid/router.py", line 242, in __call__
api_1 |     response = self.invoke_subrequest(request, use_tweens=True)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/pyramid-1.5.8-py2.7.egg/pyramid/router.py", line 217, in invoke_subrequest
api_1 |     response = handle_request(request)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/pyramid-1.5.8-py2.7.egg/pyramid/tweens.py", line 46, in excview_tween
api_1 |     response = view_callable(exc, request)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/pyramid-1.5.8-py2.7.egg/pyramid/config/views.py", line 385, in viewresult_to_response
api_1 |     result = view(context, request)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/pyramid-1.5.8-py2.7.egg/pyramid/tweens.py", line 21, in excview_tween
api_1 |     response = handler(request)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/pyramid-1.5.8-py2.7.egg/pyramid/router.py", line 163, in handle_request
api_1 |     response = view_callable(context, request)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/pyramid-1.5.8-py2.7.egg/pyramid/config/views.py", line 596, in __call__
api_1 |     return view(context, request)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/pyramid-1.5.8-py2.7.egg/pyramid/config/views.py", line 329, in attr_view
api_1 |     return view(context, request)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/pyramid-1.5.8-py2.7.egg/pyramid/config/views.py", line 305, in predicate_wrapper
api_1 |     return view(context, request)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/pyramid-1.5.8-py2.7.egg/pyramid/config/views.py", line 245, in _secured_view
api_1 |     return view(context, request)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/pyramid-1.5.8-py2.7.egg/pyramid/config/views.py", line 355, in rendered_view
api_1 |     result = view(context, request)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/pyramid-1.5.8-py2.7.egg/pyramid/config/views.py", line 501, in _requestonly_view
api_1 |     response = view(request)
api_1 |   File "/usr/local/lib/python2.7/dist-packages/cornice/service.py", line 571, in wrapper
api_1 |     response = view_()
api_1 |   File "/usr/local/lib/python2.7/dist-packages/caliopen/api/message/thread.py", line 33, in get
api_1 |     thread_id = int(self.request.matchdict.get('thread_id'))
api_1 | ValueError: invalid literal for int() with base 10: '4078e616-390c-478a-abb2-dddaae7c10f0'
```

---

PS: no problem with `/threads/{id}/messages`

**Comment by gdchamal** [comment]
fixed with 1e3cb28e75ef1f5f07fa6e14d134354e11ec7d95 in caliopen.api.message

**Comment by iamdey** [comment]
`CaliOpen/caliopen.api.message@1e3cb28e75ef1f5f07fa6e14d134354e11ec7d95`

fixed CaliOpen/caliopen.api.message@1e3cb28e75ef1f5f07fa6e14d134354e11ec7d95
